test/parallel/test-fs-watchfile.js

```diff
@@ -63,3 +63,25 @@ fs.watchFile(enoentFile, {interval: 0}, common.mustCall(function(curr, prev) {
     fs.unwatchFile(enoentFile);
   }
 }, 2));
+
+// Watch events should callback with a filename
+if (common.isLinux || common.isOSX) {
+  const dir = common.tmpDir + '/watch';
+
+  fs.mkdir(dir, common.mustCall(function(err) {
+    assert(!err);
```

Comment:
If you could use `assert.ifError` instead. It will rethrow the error so we know what went wrong.
`assert(!err)` will just say `true != false`